### PR TITLE
[DO NOT MERGE] Fix VM network config for /32 IP addresses

### PR DESCRIPTION
**Pull request by SlothOfAnarchy**

### CODE REVIEW

File: templates/interfaces.j2 [hunk 1]
```diff
@@ -6,9 +6,10 @@ auto eth0
 iface eth0 inet {% if vm.storage_type|default(xen_vman_default_storage_type) == "iscsi" %}static{% else %}manual{% endif %}
 
 	address {{ vm.connection_ip|default(vm['interfaces'][0]['ip'])|ipaddr('address') }}
-	network {{ vm.connection_ip|default(vm['interfaces'][0]['ip'])|ipaddr('network') }}
 	netmask {{ vm.connection_ip|default(vm['interfaces'][0]['ip'])|ipaddr('netmask') }}
+	{% if vm.connection_ip|default(vm['interfaces'][0]['ip'])|ipaddr('broadcast') is not None %}
```
Comment (haslersn) [inline templates/interfaces.j2]:
```suggestion
{% if vm.connection_ip|default(vm['interfaces'][0]['ip'])|ipaddr('broadcast') is not None %}
```

Comment (haslersn) [inline templates/interfaces.j2]:
```suggestion
{% if vm.connection_ip|default(vm['interfaces'][0]['ip'])|ipaddr('broadcast') is not None %}
```

**Comment by haslersn** [comment]
Is this ready?

**Comment by haslersn** [comment]
Is this ready?

**Comment by SlothOfAnarchy** [comment]
Yes, I tested it when we created helfer03 and helfer04.

**Pull request merged**